tests/ops/test_cv_ops.py

```diff
@@ -40,6 +41,26 @@ def test_rotation_heisenberg(self, phi):
         )
         assert np.allclose(matrix, true_matrix)
 
+    @pytest.mark.parametrize("op,size", [
+            (cv.Squeezing(0.123, 0.456, wires=1), 3),
```

Comment:
I suggest also testing angles phi such that phi + pi > 2pi to make sure that the modulo 2 pi is working as intended. I'm not 100% sure myself that adding pi is the way to go

Reply:
Added